web-admin/src/features/dashboards/share/ShareDashboardButton.svelte

```diff
@@ -50,3 +50,9 @@
     </Tabs>
   </PopoverContent>
 </Popover>
+
+<style lang="postcss">
+  h3 {
```

Comment:
why not tailwind style use `font-semibold` ?

Reply:
This is applying that Tailwind class. It's just separating the styles from the markup, which we've started to do across the app & IMO is a bit cleaner. But you're right that in this component, the rest of the styling is embedded in the markup, so it's not necessary.